client/tx/aux_builder.go

```diff
@@ -209,9 +219,10 @@ func (b *AuxTxBuilder) GetSignBytes() ([]byte, error) {
 			})
 
 			auxBody := &txv1beta1.TxBody{
-				Messages:      body.Messages,
-				Memo:          body.Memo,
-				TimeoutHeight: body.TimeoutHeight,
+				Messages:         body.Messages,
+				Memo:             body.Memo,
+				TimeoutHeight:    body.TimeoutHeight,
+				TimeoutTimestamp: body.TimeoutTimestamp,
```

Comment:
it looks like in the `flags` file we are replacing `TimeoutHeight` with `TimeoutTimestamp` but here we allow the use of both?

Maybe we still want to include the flag for a timeout height as well?

Reply:
the original intent was to introduce a small break to the client to push them towards using the better design of timestamp timeouts. https://github.com/cosmos/cosmos-sdk/pull/20870/files#r1686279283.

which makes sense to me - however im unsure if this is a substantial break to make or not.

Reply:
I think we should support all available flags since people might be relying on them and we cannot remove them from the proto anyways

Reply:
updated [restore timeout-height flag and factory support](https://github.com/cosmos/cosmos-sdk/pull/23734/commits/af20c8186b11e0465ba34491c6e89ce002de55a5)